File: src/main/java/jp/artan/japanesefoodmod/raw_materials/seaweed/RawMaterialSeaweed.java
package jp.artan.japanesefoodmod.raw_materials.seaweed;

import jp.artan.japanesefoodmod.common.seaweeds.CustomSeaweeds;
import jp.artan.japanesefoodmod.common.world.seaweed.WorldGenSeaweed;
import jp.artan.japanesefoodmod.raw_materials.food.RawMaterialFood;

public class RawMaterialSeaweed extends CustomSeaweeds {
    public RawMaterialSeaweed(String name, int amount, float saturation, int chance, WorldGenSeaweed worldGen) {
        super(new RawMaterialFood(name, amount, saturation), new RawMaterialSeaweedBlock(name, chance), worldGen);
    }
}

File: src/main/java/jp/artan/japanesefoodmod/raw_materials/seaweed/RawMaterialSeaweed.kt
package jp.artan.japanesefoodmod.raw_materials.seaweed

import jp.artan.japanesefoodmod.common.seaweeds.CustomSeaweeds
import jp.artan.japanesefoodmod.common.world.seaweed.WorldGenSeaweed
import jp.artan.japanesefoodmod.raw_materials.food.RawMaterialFood

class RawMaterialSeaweed(name: String, amount: Int, saturation: Float, chance: Int, worldGen: WorldGenSeaweed) :
        CustomSeaweeds(RawMaterialFood(name, amount, saturation), RawMaterialSeaweedBlock(name, chance), worldGen) {

}
